src/LinAlg/SAM.C

```diff
@@ -944,3 +944,25 @@ bool SAM::getNodalReactions (int inod, const Vector& rf, Vector& nrf) const
 
   return haveRF;
 }
+
+
+IntSet SAM::getEquations(char dofType, int dof) const
```

Comment:
det er to grunner til å bruke set;
1) vi trenger unike entries da result.size() brukes utenfor (kunne blitt håndert av std::sort etterfulgt av stq::unique)
2) oppretting av blokk ligningsnummer blir en flaskehals for store modeller. i den koden brukes resultatet til å besvare spørsmålet 'er ligning x i blokk y'. siden std::set::find() er kjappere enn std::find() på en vektor, og gir en kraftig tidsstraff.